package.json

```diff
@@ -28,7 +28,7 @@
     "chalk": "~1.1.1",
     "compression": "~1.6.1",
     "connect-flash": "~0.1.1",
-    "connect-mongo": "~0.8.2",
+    "connect-mongo": "~1.1.0",
```

Comment:
I'd rather drop Node 0.12 support, because this is not the only package we haven't updated and Node 0.12 isn't often the reported version our users use.